router/router.go

```diff
@@ -21,6 +21,7 @@ func New(s *server.Server) *mux.Router {
 	f := s.HandlerFunc
 	router.Methods("GET").Path("/v2/services").Handler(f(schemas, s.ServiceList))
 	router.Methods("GET").Path("/v2/service").Handler(f(schemas, s.ServiceList))
+	router.Methods("GET").Path("/v2/environments/{environment}/").Handler(f(schemas, s.ServiceList))
```

Comment:
This should be `"/v2/environments/{environment}/services"`.  Can you just change the two above and not add a third.